### ISSUE TITLE
Ambiguous cites with more than three authors

### ISSUE THREAD

**Issue by mattiasnek** [plk/biblatex#336]
I have an unexpected behavior using biblatex and references with more than three authors.

An MWE:

```
\documentclass{article}
\usepackage[backend=biber,authordate,uniquelist]{biblatex-chicago}

\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}

@article{batty2007iq,
title={IQ in early adulthood and later cancer risk: cohort study of one million Swedish men},
author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
journal={Annals of Oncology},
volume={18},
number={1},
pages={21--28},
year={2007},
publisher={European Society for Medical Oncology}
}

@article{batty2007premorbid,
title={Premorbid (early life) IQ and Later Mortality Risk: Systematic Review},
author={Batty, G. David and Deary, Ian J. and Gottfredson, Linda S.},
journal={Annals of Epidemiology},
volume={17},
number={4},
pages={278--288},
year={2007},
publisher={Elsevier}
}

@article{batty2009iq_early,
title={IQ in Early Adulthood and Mortality By Middle Age: Cohort Study of 1 Million Swedish Men},
author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
journal={Epidemiology},
volume={20},
number={1},
pages={100--109},
year={2009},
publisher={LWW}
}

@article{batty2009iq_socio,
title={IQ in Early Adulthood, Socioeconomic Position, and Unintentional Injury Mortality by Middle Age: A Cohort Study of More Than 1 Million Swedish Men},
author={Batty, G. David and Gale, Catharine R. and Tynelius, Per and Deary, Ian J. and Rasmussen, Finn},
journal={American Journal of Epidemiology},
volume={169},
number={5},
pages={606--615},
year={2009},
publisher={Oxford University Press}
}
\end{filecontents}

\addbibresource{\jobname.bib}

\begin{document}

\autocite{batty2007iq}

\autocite{batty2007premorbid}

\autocite{batty2009iq_early}

\autocite{batty2009iq_socio}

\printbibliography

\end{document}
```

Above code results in
![4lrez](https://cloud.githubusercontent.com/assets/14248511/9838678/784cf870-5a63-11e5-9be4-82d29892d591.png)
which is ambiguous for "(Batty et al. 2009)" and "(Batty, Gale, et al. 2009)".

However, if I comment `\autocite{batty2007iq}` or `\autocite{batty2009iq_early}` the cites are unambiguous. So, if I comment `\autocite{batty2007iq}`, the result is:
![dejxd](https://cloud.githubusercontent.com/assets/14248511/9838704/2a83d43c-5a64-11e5-9d32-59cc6eef90ea.png)

What I expect is

```
(Batty, Wennerstad, et al. 2007)
(Batty, Deary, and Gottfredson 2007)
(Batty, Wennerstad, et al. 2009)
(Batty, Gale, et al. 2009)
```

Is this a bug in biblatex?

**Comment by plk** [comment]
Can you ask that on tex.stackexchange.com? It’s an interesting question and it would be better to have the discussion public.

PK

> On 13 Sep 2015, at 10:14 pm, mattiasnek <notifications@github.com> wrote:
> 
> I have an unexpected behavior using biblatex and references with more than three authors.
> 
> An MWE:
> 
> \documentclass{article}
> \usepackage[backend=biber,authordate,uniquelist]{biblatex-chicago}
> 
> \usepackage{filecontents}
> \begin{filecontents}{\jobname.bib}
> 
> @article{batty2007iq,
> title={IQ in early adulthood and later cancer risk: cohort study of one million Swedish men},
> author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
> journal={Annals of Oncology},
> volume={18},
> number={1},
> pages={21--28},
> year={2007},
> publisher={European Society for Medical Oncology}
> }
> 
> @article{batty2007premorbid,
> title={Premorbid (early life) IQ and Later Mortality Risk: Systematic Review},
> author={Batty, G. David and Deary, Ian J. and Gottfredson, Linda S.},
> journal={Annals of Epidemiology},
> volume={17},
> number={4},
> pages={278--288},
> year={2007},
> publisher={Elsevier}
> }
> 
> @article{batty2009iq_early,
> title={IQ in Early Adulthood and Mortality By Middle Age: Cohort Study of 1 Million Swedish Men},
> author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
> journal={Epidemiology},
> volume={20},
> number={1},
> pages={100--109},
> year={2009},
> publisher={LWW}
> }
> 
> @article{batty2009iq_socio,
> title={IQ in Early Adulthood, Socioeconomic Position, and Unintentional Injury Mortality by Middle Age: A Cohort Study of More Than 1 Million Swedish Men},
> author={Batty, G. David and Gale, Catharine R. and Tynelius, Per and Deary, Ian J. and Rasmussen, Finn},
> journal={American Journal of Epidemiology},
> volume={169},
> number={5},
> pages={606--615},
> year={2009},
> publisher={Oxford University Press}
> }
> \end{filecontents}
> 
> \addbibresource{\jobname.bib}
> 
> \begin{document}
> 
> \autocite{batty2007iq}
> 
> \autocite{batty2007premorbid}
> 
> \autocite{batty2009iq_early}
> 
> \autocite{batty2009iq_socio}
> 
> \printbibliography
> 
> \end{document}
> 
> Above code results in
> 
> which is ambiguous for "(Batty et al. 2009)" and "(Batty, Gale, et al. 2009)".
> 
> However, if I comment \autocite{batty2007iq} or \autocite{batty2009iq_early} the cites are unambiguous. So, if I comment \autocite{batty2007iq}, the result is:
> 
> 
> What I expect is
> 
> (Batty, Wennerstad, et al. 2007)
> (Batty, Deary, and Gottfredson 2007)
> (Batty, Wennerstad, et al. 2009)
> (Batty, Gale, et al. 2009)
> 
> Is this a bug in biblatex?
> 
> —
> Reply to this email directly or view it on GitHub.
> 

--
Dr Philip Kime

**Comment by mattiasnek** [comment]
Oh, forgot to mention that: http://tex.stackexchange.com/questions/267113/unexpected-behavior-with-biblatex-chicago-and-references-with-more-than-three-au

/Mattias

Från: plk [mailto:notifications@github.com]
Skickat: den 13 september 2015 22:17
Till: plk/biblatex
Kopia: Mattias Öhman
Ämne: Re: [biblatex] Ambiguous cites with more than three authors (#336)

Can you ask that on tex.stackexchange.com? It’s an interesting question and it would be better to have the discussion public.

PK

> On 13 Sep 2015, at 10:14 pm, mattiasnek <notifications@github.com<mailto:notifications@github.com>> wrote:
>
> I have an unexpected behavior using biblatex and references with more than three authors.
>
> An MWE:
>
> \documentclass{article}
> \usepackage[backend=biber,authordate,uniquelist]{biblatex-chicago}
>
> \usepackage{filecontents}
> \begin{filecontents}{\jobname.bib}
>
> @article{batty2007iq,
> title={IQ in early adulthood and later cancer risk: cohort study of one million Swedish men},
> author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
> journal={Annals of Oncology},
> volume={18},
> number={1},
> pages={21--28},
> year={2007},
> publisher={European Society for Medical Oncology}
> }
>
> @article{batty2007premorbid,
> title={Premorbid (early life) IQ and Later Mortality Risk: Systematic Review},
> author={Batty, G. David and Deary, Ian J. and Gottfredson, Linda S.},
> journal={Annals of Epidemiology},
> volume={17},
> number={4},
> pages={278--288},
> year={2007},
> publisher={Elsevier}
> }
>
> @article{batty2009iq_early,
> title={IQ in Early Adulthood and Mortality By Middle Age: Cohort Study of 1 Million Swedish Men},
> author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
> journal={Epidemiology},
> volume={20},
> number={1},
> pages={100--109},
> year={2009},
> publisher={LWW}
> }
>
> @article{batty2009iq_socio,
> title={IQ in Early Adulthood, Socioeconomic Position, and Unintentional Injury Mortality by Middle Age: A Cohort Study of More Than 1 Million Swedish Men},
> author={Batty, G. David and Gale, Catharine R. and Tynelius, Per and Deary, Ian J. and Rasmussen, Finn},
> journal={American Journal of Epidemiology},
> volume={169},
> number={5},
> pages={606--615},
> year={2009},
> publisher={Oxford University Press}
> }
> \end{filecontents}
>
> \addbibresource{\jobname.bib}
>
> \begin{document}
>
> \autocite{batty2007iq}
>
> \autocite{batty2007premorbid}
>
> \autocite{batty2009iq_early}
>
> \autocite{batty2009iq_socio}
>
> \printbibliography
>
> \end{document}
>
> Above code results in
>
> which is ambiguous for "(Batty et al. 2009)" and "(Batty, Gale, et al. 2009)".
>
> However, if I comment \autocite{batty2007iq} or \autocite{batty2009iq_early} the cites are unambiguous. So, if I comment \autocite{batty2007iq}, the result is:
>
>
> What I expect is
>
> (Batty, Wennerstad, et al. 2007)
> (Batty, Deary, and Gottfredson 2007)
> (Batty, Wennerstad, et al. 2009)
> (Batty, Gale, et al. 2009)
>
> Is this a bug in biblatex?
>
> —
> Reply to this email directly or view it on GitHub.
>

--
Dr Philip Kime

**Comment by plk** [comment]
That is the expected behaviour as the disambiguation is intended to provide unique references into the bibliography. In such circumstances, mincite/bibnames is the right way to deal with this.

**Comment by mattiasnek** [comment]
Ok.

Since one couldn't really know what "(Batty et al. 2009)" refers to in the bibliography without knowing that a "(Batty, Gale, et al. 2009)" exists, the result is ambiguous for the human reader. If this is the expected behavior, would it be possible to implement a package option which gives my "expected behavior"? Otherwise one has to use mincitenames on a per-entry basis in a way that becomes really cumbersome with bibliographies with about 60-70 entries and a lot of authors. Also, I guess a journal would never accept cites with the current "standard" behavior.

And I really think it's quite unintuitive that commenting \autocite{batty2007iq} affects the display of \autocite{batty2009iq_early}. If "(Batty et al. 2009)" is fine when I also have \autocite{batty2007iq}, why not when it's not there?

**Comment by plk** [comment]
Well there is really an option:

```
\usepackage[backend=biber,authordate,uniquelist,mincitenames=2]{biblatex-chicago}
```
This is quite specific to your bibliography and a global mincitenames is meant for this sort of situation.

**Comment by moewew** [comment]
To me it seems that @mattiasnek objects to the behaviour shown in

```
\documentclass{article}
\usepackage[backend=biber,authordate,uniquelist]{biblatex-chicago}

\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}

@article{batty2007iq,
title={IQ in early adulthood and later cancer risk: cohort study of one million Swedish men},
author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
journal={Annals of Oncology},
volume={18},
number={1},
pages={21--28},
year={2007},
publisher={European Society for Medical Oncology}
}

@article{batty2007premorbid,
title={Premorbid (early life) IQ and Later Mortality Risk: Systematic Review},
author={Batty, G. David and Deary, Ian J. and Gottfredson, Linda S.},
journal={Annals of Epidemiology},
volume={17},
number={4},
pages={278--288},
year={2007},
publisher={Elsevier}
}

@article{batty2009iq_early,
title={IQ in Early Adulthood and Mortality By Middle Age: Cohort Study of 1 Million Swedish Men},
author={Batty, G. David and Wennerstad, Karin Modig and Smith, George Davey and Gunnell, David and Deary, Ian J. and Tynelius, Per and Rasmussen, Finn},
journal={Epidemiology},
volume={20},
number={1},
pages={100--109},
year={2009},
publisher={LWW}
}

@article{batty2009iq_socio,
title={IQ in Early Adulthood, Socioeconomic Position, and Unintentional Injury Mortality by Middle Age: A Cohort Study of More Than 1 Million Swedish Men},
author={Batty, G. David and Gale, Catharine R. and Tynelius, Per and Deary, Ian J. and Rasmussen, Finn},
journal={American Journal of Epidemiology},
volume={169},
number={5},
pages={606--615},
year={2009},
publisher={Oxford University Press}
}
\end{filecontents}

\addbibresource{\jobname.bib}

\begin{document}
\begin{refsection}
\section{With \texttt{batty2007iq}}
Possibly ambiguous citations

\autocite{batty2007iq}

\autocite{batty2007premorbid}

\autocite{batty2009iq_early}

\autocite{batty2009iq_socio}
\end{refsection}

\begin{refsection}
\section{Without \texttt{batty2007iq}}
Unambiguous

\autocite{batty2007premorbid}

\autocite{batty2009iq_early}

\autocite{batty2009iq_socio}
\end{refsection}
\end{document} 
```
Where we get

> <h4>1 With <code>batty2007iq</code></h4>
Possibly ambiguous citations
(Batty et al. 2007)
(Batty, Deary, and Gottfredson 2007)
(Batty et al. 2009)
(Batty, Gale, et al. 2009)
2 <h4>Without <code>batty2007iq</code></h4>
Unambiguous
(Batty, Deary, and Gottfredson 2007)
(Batty, Wennerstad, et al. 2009)
(Batty, Gale, et al. 2009)


The first refsection gives exactly what we would expect, but as soon as another citation with the same name list (`batty2007iq`) is added in the second refsection, the citations are not clearly unambiguous any more.

**Comment by mattiasnek** [comment]
@plk "(First Lastname et al. Year)" is perfectly fine most of the time, e.g. when there can be no confusion (and as far as I understand what the Chicago style requires, see for example http://www.chicagomanualofstyle.org/tools_citationguide.html ). It's only when needed more than one name should be printed in the cite if there are more than three authors.

This situation is very common in epidemiology and medicine, where we usually cite lots of papers and most of them have more than three authors (some probably with the same first author).

**Comment by plk** [comment]
Ah, in that case this is something for the biblatex-chicago style author - I had to implement similar rules for the biblatex-apa style.

**Comment by moewew** [comment]
@plk I really hope that I'm not getting on your nerves here. But is the behaviour shown in my MWE a few posts further up really the intended behaviour? And if so can you explain why? It just seesm so weird that adding anoter entry with the same name list would change the name disambiguation feature. It requires much more cognitive load from the reader to identify citations in the second case whereas in the first case with only one citation everything works fine.

**Comment by plk** [comment]
You are quite right, this is a bug. I can see where it happens. Looking at it.

**Comment by plk** [comment]
This should be fixed now in `biber` 2.7 DEV version.

**Comment by moewew** [comment]
Indeed. Thank you very much.
